### PR TITLE
Icons for Model Setting, Selct, and Category

### PR DESCRIPTION
**Pull request by olliw42**
The currently used icons for Model Setting and Model Selecting are quite strange and confusing and inconsistent.

This PR approaches this by 
* adding icon enums ICON_MODELSELECT and ICON_MODELSELECT_CATEGORY to be used in the model select menu page
* using ICON_MODEL instead of ICON_RADIO_TOOLS for Model Settings in the main menu, as this is the one also used in the model settings menu

For the icons of  ICON_MODELSELECT and ICON_MODELSELECT_CATEGORY I have used some from what is already available and what is not totally strange. However, for these two icons someone with some graphics design capabilities (= not me LOL) SHOULD create and add proper new icons !!!

I know I should add screenshots, but ...

### CODE REVIEW

**Comment by pfeerick** [comment]
Oh, very cice catch! :) 

In other words, the icon shown when in the model settings screen was being shown as the 'select model' icon when in the main menu. Not confusing at all! 😆

File: radio/src/gui/colorlcd/menus.h [hunk 1]
```diff
@@ -54,6 +54,8 @@ enum MenuIcons {
   ICON_MODEL_SPECIAL_FUNCTIONS,
   ICON_MODEL_LUA_SCRIPTS,
   ICON_MODEL_TELEMETRY,
+  ICON_MODELSELECT,
+  ICON_MODELSELECT_CATEGORY,
```
Comment (raphaelcoeffic) [inline radio/src/gui/colorlcd/menus.h:58]: I'd like to buy a `_` to make it look like the others.

Comment (olliw42) [reply radio/src/gui/colorlcd/menus.h:58]: it's actually the reason I did not add the _
their general pattern is ICON MENU CATEGORY
MODEL_SELECT, would it be the category select in the model menu, or not?

Comment (raphaelcoeffic) [reply radio/src/gui/colorlcd/menus.h:58]: Just change it to `ICON_MODEL_SELECT` & `ICON_MODEL_SELECT_CATEGORY`.

**Comment by olliw42** [comment]
the other PR has been merged, so done, so I close

**Pull request closed without merging**